src/routes/transactions/mappers/common/safe-app-info.mapper.ts

```diff
@@ -0,0 +1,52 @@
+import { Inject, Injectable, Logger } from '@nestjs/common';
+import { SafeAppsRepository } from '../../../../domain/safe-apps/safe-apps.repository';
+import { ISafeAppsRepository } from '../../../../domain/safe-apps/safe-apps.repository.interface';
+import { MultisigTransaction } from '../../../../domain/safe/entities/multisig-transaction.entity';
+import { SafeAppInfo } from '../../entities/safe-app-info.entity';
+
+@Injectable()
+export class SafeAppInfoMapper {
+  private readonly logger = new Logger(SafeAppInfoMapper.name);
+  private static readonly IPFS_URL = 'ipfs.io';
+  private static readonly CF_IPFS_URL = 'cloudflare-ipfs.com';
+
+  constructor(
+    @Inject(ISafeAppsRepository)
+    private readonly safeAppsRepository: SafeAppsRepository,
+  ) {}
+
+  async mapSafeAppInfo(
+    chainId: string,
+    transaction: MultisigTransaction,
+  ): Promise<SafeAppInfo | null> {
+    const originUrl = this.getOriginUrl(transaction);
+    if (!originUrl) return null;
+
+    const [safeApp] = await this.safeAppsRepository.getSafeApps(
+      chainId,
+      undefined,
+      originUrl,
+    );
+    if (!safeApp) {
+      this.logger.error(
```

Comment:
This should not be an error in my opinion.

Maybe `info` or `warning`. Not finding a Safe App is part of what we consider to be "normal".

Reply:
I'm changing it to `warn` in [d9abbce](https://github.com/5afe/safe-client-gateway-nest/pull/236/commits/d9abbce9dd91fd30e603198f8ea696748569eb1a), I thought not finding a Safe App could be considered an error. Anyway, ping me if you feel `info` is more appropriate here.